portallauncher.py

```diff
@@ -27,7 +27,6 @@
 import errno
 import logging
 import os
-import sys
```

Comment:
This renaming patch appears to contain a code change? It would ideally be committed separately.

Reply:
Not relevant anymore: the new PR won't include this file at all